TestProject/src/rules/BaseRule.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace GameOfLife\Rules;
+
+use GameOfLife\options\Getopt;
+use GameOfLife\cellularAutomat\Field;
+
+/**
+ * API-class for additional rules.
+ */
+abstract class BaseRule
+{
+    /**
+     * @param Field $_field The cell which is going to be checked.
```

Comment:
Here you could place a hint for the developer and explain how the new state could be calculated. For example: What methods does the Field provide that are helpful to implement that?